core/src/main/java/edu/ucr/cs/riple/core/metadata/graph/ConflictGraph.java

```diff
@@ -74,7 +74,7 @@ public Node addNodeToVertices(Fix fix) {
    * in the same group. A greedy algorithm is used to find the solution.
    */
   // TODO: Remove SuppressWarnings below later.
-  @SuppressWarnings("JdkObsolete")
+  @SuppressWarnings({"JdkObsolete"})
```

Comment:
Left-over change? Curly brackets shouldn't be needed if only one checker is being suppressed. Though I guess this works either way :)

Reply:
Yes this is a left-over, thank you for noticing this, removed it. c82365feb4aef9a2988f3f9a773ad20afe55d2e4